tests/test_configs/autoguard/harm_detection/config.yml

```diff
@@ -0,0 +1,13 @@
+models:
```

Comment:
why the same config file multiple times for each guardrail? cant they share one model config? this looks error-prone

Reply:
resolved in prev commits